membrane_curvature/base.py

```diff
@@ -98,17 +98,22 @@ class MembraneCurvature(AnalysisBase):
     The respective 2D curvature plots can be obtained using the `matplotlib` package for
     data visualization via `imshow`. We recommend using the `gaussian` interpolation. 
 
-    A simple plot using `imshow` can be obtained by
+    A simple plot using `imshow` can be obtained by::
```

Comment:
Maybe remove this text from the Python doc string and move it to a dedicated reST file where you
1. show a real example with data
2. include actual images

That makes the example much more impactful.